rollup-bridge-contracts/hardhat.config.ts

```diff
@@ -97,7 +97,7 @@ const config: HardhatUserConfig = {
     },
     nil: {
       url: process.env.NIL_RPC_ENDPOINT,
-      accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : [],
+      accounts: process.env.NIL_PRIVATE_KEY ? [process.env.NIL_PRIVATE_KEY] : [],
```

Comment:
Does `PRIVATE_KEY` environment variable used by any another scripts?

Reply:
@akokoshn this is used in the function to create a new smart-account
we are in midst of getting rid of this variable and create new key-pair for unit tests